logging/google/cloud/logging/_http.py

```diff
@@ -319,7 +325,9 @@ def sink_update(self, project, sink_name, filter_, destination):
             'filter': filter_,
             'destination': destination,
         }
-        return self.api_request(method='PUT', path=target, data=data)
+        query_params = {'uniqueWriterIdentity': unique_writer_identity}
```

Comment:
@tseaver Is passing an explicit false somehow different than the "implicit false" of sending nothing at all?

Reply:
For [update](https://cloud.google.com/logging/docs/reference/v2/rest/v2/projects.sinks/update#body.QUERY_PARAMETERS.unique_writer_identity), the docs say:

> When updating a sink, the effect of this field on the value of writerIdentity in the updated sink depends on both the old and new values of this field:
>
> - If the old and new values of this field are both false or both true, then there is no change to the sink's writerIdentity.
 > - If the old value is false and the new value is true, then writerIdentity is changed to a unique service account.
 > - It is an error if the old value is true and the new value is set to false or defaulted to false.

Reply:
Which is a bogus API choice, but the answer is "no, no difference."